### ISSUE TITLE
Have plugins as dependencies (not peerDependencies)

### ISSUE THREAD

**Issue by mightyiam** [standard/eslint-config-standard-with-typescript#97]
Blocked by https://github.com/eslint/eslint/issues/3458.

**Comment by Divlo** [comment]
Possibly fixed thanks to https://www.npmjs.com/package/@rushstack/eslint-patch

**Comment by mightyiam** [comment]
> Possibly fixed thanks to https://www.npmjs.com/package/@rushstack/eslint-patch

Maybe, but we are downstream to `eslint-config-standard` and should only use this thingy if upstream uses it, I suppose.

**Comment by meyfa** [comment]
https://github.com/eslint/eslint/issues/3458 is now available experimentally, but will likely require significant changes to eslint-config-standard-with-typescript

**Comment by mightyiam** [comment]
@meyfa . Wow. Thank you for letting us know.

**Comment by rostislav-simonik** [comment]
Closing in favor of https://github.com/standard/standard/issues/1871
